src/Symfony/Component/Polyfill/Php55.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Polyfill;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @internal
+ */
+class Php55
+{
+    public static function boolval($val)
+    {
+        return (bool) $val;
+    }
+
+    public static function json_last_error_msg()
+    {
+        switch (json_last_error()) {
+            case JSON_ERROR_NONE: return "No error";
+            case JSON_ERROR_DEPTH: return "Maximum stack depth exceeded";
+            case JSON_ERROR_STATE_MISMATCH: return "State mismatch (invalid or malformed JSON)";
+            case JSON_ERROR_CTRL_CHAR: return "Control character error, possibly incorrectly encoded";
+            case JSON_ERROR_SYNTAX: return "Syntax error";
+            case JSON_ERROR_UTF8: return "Malformed UTF-8 characters, possibly incorrectly encoded"; // Since 5.3.3
+            case JSON_ERROR_RECURSION: return "Recursion detected"; // Since 5.5.0
+            case JSON_ERROR_INF_OR_NAN: return "Inf and NaN cannot be JSON encoded"; // Since 5.5.0
+            case JSON_ERROR_UNSUPPORTED_TYPE: return "Type is not supported"; // Since 5.5.0
+            default: return "Unknown error";
+        }
+    }
+
+    public static function opcache_invalidate($file, $force = false)
+    {
+        return self::opcache_reset();
+    }
+
+    public static function opcache_reset()
```

Comment:
polyfilling opcache methods by applying changes in other accelerators looks really weird to me.